docs/book/src/clusterctl/commands/generate-cluster.md

```diff
@@ -21,23 +30,23 @@ kubectl apply -f my-cluster.yaml
 
 ### Selecting the infrastructure provider to use
 
-The `clusterctl generate cluster` command uses smart defaults in order to simplify the user experience; in the example above,
+The `clusterctl generate cluster` command uses smart defaults in order to simplify the user experience; in the example below,
```

Comment:
Thank you for clarifying because I realize now I misunderstood what it meant before. 

I agree with adding your suggestion. Additionally I suggest mentioning that you need to have a configured management cluster in your current context. The command didn't work for me until I ran `clusterctl init --infrastructure vcluster` in my kind cluster. I hope that explanation makes sense.